src/components/__tests__/ListCardPost.cy.js

```diff
@@ -142,41 +103,51 @@ describe('<ListCardPost>', () => {
       cy.viewport('iphone-6');
     });
 
-    it('renders title', () => {
-      cy.window().then(() => {
-        cy.dataCy('card-list-post-title')
-          .should('have.css', 'font-size', '20px')
-          .and('have.css', 'font-weight', '500')
-          .and('have.color', black)
-          .and('contain', title)
-          .then(($title) => {
-            expect($title.text()).to.equal(title);
-          });
-      });
-    });
+    coreTests();
 
-    it('renders correct number of items', () => {
-      cy.window().then(() => {
-        cy.dataCy('card-list-post-item').should('have.length', 5);
-      });
+    it('renders full width button container', () => {
+      cy.dataCy('card-list-post-buttons').should('be.visible');
+      cy.testElementPercentageWidth(cy.dataCy('card-list-post-buttons'), 100);
     });
 
-    it('renders button container', () => {
-      cy.window().then(() => {
-        cy.dataCy('card-list-post-buttons').should('be.visible');
-        cy.testElementPercentageWidth(cy.dataCy('card-list-post-buttons'), 100);
-      });
+    it('renders full width button', () => {
+      cy.dataCy('card-list-post-button')
+        .should('be.visible')
+        .and('have.css', 'border-color', hexToRgb('#212121'))
```

Comment:
Hex color '#212121' repeat and should not be simple black color?

Reply:
Color was taken from wireframe. Changes are that it will change in the final iteration of the design anyway.
Unfortunately, the 'border-color' check does not accept hex color as it compares the exact CSS value. However, I replaced `#212121` with the Quasar color palette `grey-10` shade.